### DISCUSSION TITLE
number of layers

### DISCUSSION THREAD

**Discussion by spgerenday** [giorgk/ichnos#6] [Q&A] [answered]
Does Nlay in the well file need to be the same as the number of layers in the model (4 for C2VSimFG), or can it be any number by which the number of particles is divisible?

**Comment by giorgk** [comment, marked as answer]
No it has nothing to do with the number of model layers. 
I have updated the explanation about the distribution of particles around wells. I hope it gets clear now
https://github.com/giorgk/ichnos/wiki/File-formats#wellfile
